lib/upipe-srt/upipe_srt_receiver.c

```diff
@@ -617,11 +622,19 @@ static void upipe_srt_receiver_timer(struct upump *upump)
             old = now;
         }
 
-        if (--upipe_srt_receiver->buffered == 0) {
-            upipe_warn_va(upipe, "Exhausted buffer");
-            upipe_srt_receiver->expected_seqnum = UINT64_MAX;
-            upipe_srt_receiver->last_output_seqnum = UINT64_MAX;
-        }
+        upipe_srt_receiver->buffered--;
+    }
+
+    /* At low bitrates and VBR video the buffer may momentarily exhaust. 
```

Comment:
Whitespace at the end of line triggers coding style buildbot

Reply:
I will remove this commit